resources/measures/upgrade_hvac_enable_ideal_air_loads/measure.rb

```diff
@@ -93,10 +93,17 @@ def run(model, runner, user_arguments)
 
     # remove any EMS objects tied to ground HX; these will fail model since HVAC has been removed
     # get all EMS objects in the model
-    ems_objects = model.getEnergyManagementSystemSensors.to_a + model.getEnergyManagementSystemActuators.to_a + model.getEnergyManagementSystemPrograms.to_a + model.getEnergyManagementSystemProgramCallingManagers.to_a + model.getEnergyManagementSystemInternalVariables.to_a
-
-    # Filter EMS objects that contain "pvav"
-    ems_objects_to_remove = ems_objects.select { |ems_object| ems_object.name.to_s.include?('Ground_HX') }
+    ems_objects = model.getEnergyManagementSystemSensors.to_a +
+                  model.getEnergyManagementSystemActuators.to_a +
+                  model.getEnergyManagementSystemPrograms.to_a +
+                  model.getEnergyManagementSystemProgramCallingManagers.to_a +
+                  model.getEnergyManagementSystemInternalVariables.to_a
+
+    # Filter EMS objects that contain any of the strings in the list
+    ems_string_to_remove = ['Ground_HX', 'PVAV']
```

Comment:
Looking thru my collection of ComStock model files, I found one with EMS code generated by `air_loop_havc_enable_optimum_start` in Standards, applied to a VAV system. You could capture both by changing 'PVAV'  here to 'VAV'.